### PR TITLE
Created .travis.yml

### PR DESCRIPTION
**Pull request by DaniilRomanov**
Created `.travis.yml` file for testing with Travis-CI, currently for s390x architecture only.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - focal
+arch:
+  - s390x
+language: 
+  - shell
+install: 
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+  - echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+  - sudo apt-get update
+  - sudo apt-get install docker-ce
```
Comment (krishvoor) [inline .travis.yml:12]: Can we use this instead
```
services:
  - docker
```

**Review by krishvoor — COMMENTED** [review]
Minor nit

File: .travis.yml (hunk 1 above)
Comment (DaniilRomanov) [reply .travis.yml:12]: Unfortunately no. It is already defined for Apisonator to build separate docker image for CI (`Dockerfile.ci`), in this image it compiles specific version of PostgreSQL, and there is a problem with PostgreSQL `./configure` inside an image building and default version of Docker on z at Travis. When I used Docker from Travis environment (version 18), `./configure` stops with an error that it doesn't recognize the shell and can't proceed. When I switched to installing Docker (current version is 20) by official Docker instruction - `./configure` works without errors.

**Review by krishvoor — APPROVED** [review]
lgtm

File: .travis.yml (hunk 1 above)
Comment (krishvoor) [reply .travis.yml:12]: Thanks for the explanation!

**Pull request merged**